Library/Homebrew/formula.rb

```diff
@@ -2933,7 +2933,9 @@ def pour_bottle?(only_if: nil, &block)
         lambda do |_|
           on_macos do
             T.cast(self, PourBottleCheck).reason(+<<~EOS)
-              The bottle needs the Apple Command Line Tools to be installed.
+              The bottle needs the standalone Apple Command Line Tools to be installed.
+              The full version of Xcode is not compatible with this bottle.
+              You will need to install the standalone version.
```

Comment:
This line doesn't add anything over the first sentence – it's pretty much just repeating the same thing again.

Reply:
I have removed the line, thank you for your suggestion.